lib/fakes3/server.rb

```diff
@@ -477,6 +477,11 @@ def normalize_request(webrick_req)
       return s_req
     end
 
+    def gral_strip(string, chars)
```

Comment:
Code looks good, but can you come up with a better name for this or add a comment for how this is transforming the input string?